cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -234,6 +262,25 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
         }
     };
 
+    private onChangeBlockState = (event:string, thresholdWasModified: boolean):void => {
+        const {
+            activeControl, toolsBlockerState, onSwitchBlockMode, canvasInstance,
+        } = this.props;
+        if ([ActiveControl.OPENCV_TOOLS].includes(activeControl) && event === 'keyup' && !thresholdWasModified) {
```

Comment:
We are passing it as an argument because this callback may be used not only by intelligent scissors but also with interactors(And probably with another tools in the future). So another tools may not have threshold and do not depend on it, but only on keyup/keydown events